trimmer_decrator.rb

```diff
@@ -0,0 +1,11 @@
+# require_relative 'base_decrator'
+
+class TrimmerDecorator < BaseDecorator
+  def correct_name
+    if @nameable.correct_name.length < 11
+      @nameable.correct_name
+    else
+      @nameable.correct_name.strip
+    end
+  end
```

Comment:
- Hi, reading the TrimmerDecorator requirements, it says that **the output of @nameable.correct_name has a maximum of 10 characters. If it's longer it should trim the word**. In testing your code, I noticed that this method is not working. Please work on it again and submit it for re-review.
- Can you remove commented code seen above? because it makes the code look dirty 
![Capture](https://github.com/AliShahSafdari/school_library/assets/63972670/9716b183-809b-42bc-bee7-0f95a9eb6e60)

Reply:
Thanks